src/gfx/shaders/AO.frag

```diff
@@ -49,6 +48,13 @@ void main() {
   vec3 viewPos = ViewPosFromDepth(vUv);
   // remap coordinates to prevent noise exture rescale
   vec2 vUvNoise = vUv / srcTexelSize * noiseTexelSize;
+  vec4 normalData = texture2D(normalTexture, vUv);
+  // return for background fragments (their normals are zero vectors)
+  if (length(normalData.rgb) < 0.000000001) {
```

Comment:
As I know  we use float type here, why do you compare with the 9th digit after a point?

Reply:
Sorry, for confusing you.
There was no specific reason to use in comparison 9th digits after a point. Changed it on usual 0.0000001